gradle/versions.gradle

```diff
@@ -137,5 +137,8 @@ dependencyManagement {
        \-- org.apache.tuweni:tuweni-net:2.3.1
      */
     dependency 'commons-net:commons-net:3.9.0'
+
+    // used in tests to assert log message
+    dependency 'de.neuland-bfi:assertj-logging-log4j:0.5.0'
```

Comment:
Nice solution!
Should we limit this to test scope in the deps?

Reply:
I don't think we can limit the dependency declaration to test or other scopes here. It is declared as testImplementation in common's build.gradle though.